gladier/client.py

```diff
@@ -602,15 +605,20 @@ def run_flow(self, flow_input=None, use_defaults=True):
                     log.info('Initiating new login for dependent scope change')
                     self.login(requested_scopes=[cfg_sec['flow_scope']], force=True)
                     flow = self.flows_client.run_flow(flow_id, cfg_sec['flow_scope'],
-                                                      combine_flow_input, **flow_permissions).data
+                                                      combine_flow_input, label=flow_label,
+                                                       **flow_permissions).data
```

Comment:
Can we broaden this to accept other flow kwargs besides label?

Reply:
Yes. I was just following the standards on https://globus-automate-client.readthedocs.io/en/latest/python_sdk_reference.html#globus_automate_client.flows_client.FlowsClient.run_flow

Reply:
Shall we maintain the globus standard or move label to the kwargs? I also see you have the **flow_permissions there.

Reply:
`flow_kwargs` a dict dedicated to passing args through to automate's `run_flow` would be ideal I think. That would encompass flow_permissions, flow_label, and any other flow argument they might add in later. This also allows any flow parameter Gladier passes in to be overridden by adding things on `flow_kwargs`.